### ISSUE TITLE
Server with map does not have CSS

### ISSUE THREAD

**Issue by cpkonservator** [AHAAAAAAA/PokemonGo-Map#3010]
Started my server with map which can be used on other devices (using ngrok). Here's how it looks like: http://i.imgur.com/D2Zd2UI.png

Console is working fine and has results with pokemon, gyms, etc. but this CSS issue was founded on every device (with different internet connections) I use (and friends too). Recently i've created servers based on old versions of this archive (using example.py on command line) and everything was great, map illustrated correctly.

The original command (some words were changed to ***): C:\Python27\Python.exe runserver.py -a ptc -u *** -p *** -l "Movchanskogo street 43, Mogilev" -st 7 -H 0.0.0.0 -L en -k *** -P 322

Pip is installed. Command Line works fine, no errors. I don't know what goes wrong. Here's an example of perfectly working scheme: https://www.youtube.com/watch?v=BMXsJKFYFN8

ngrok link (localhost:322 mirror): http://aff0bfee.ngrok.io/
Server: Windows 10 x64, Python 2.7

Devices used: 3 Windows 10 PCs, 2 Android phones, iPad. Same result everywhere.

I guess problem is on my computer side. But what is it?

![](http://i.imgur.com/D2Zd2UI.png)

**Comment by hack3rjack** [comment]
There were likely some dependencies not installed correctly, I'd wipe it and start from scratch paying attention to error messaging and logs

**Comment by cpkonservator** [comment]
Haven't noticed any errors during whole process. Anyway, thanks, gonna try to re-install tomorrow!

**Comment by PathToLife** [comment]
http://127.0.0.1:5000/static/dist/css/app.min.css
Make sure you can call that. I'm guessing u can't
If you can't, then congrats! You know the reason, then can start finding solution =0

Possible reasons for above: When u extracted, it didn't fully extract static folder stuff... Antivirus stopped you... You are using some custom server that's acting like htaccess.

**Comment by cpkonservator** [comment]
Yeah, I can't access. Don't have antivirus, so this strange problem is hiding somewhere =\ Will try to fix it tomorrow, thx!

**Comment by Retalyx** [comment]
try to run "grunt build" in the root folder

**Comment by mrmrblakey** [comment]
I am having the same problem, it appears that it works with Mozilla Firefox and not chrome, maybe this will help you with your problem.

**Comment by c133** [comment]
So when I start my beehive of workers the maps works for a while, then after a period of time I get what is pictured in the OP, and in the browser console the .css file request is 404ing, how do I fix this?
